### ISSUE TITLE
pfcmd node view all doesn't work

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5116]
**Describe the bug**
When you run `pfcmd node view all`,  you only see column headers. In `packetfence.log`:

```
Feb  6 09:04:58 pfcen7stable packetfence: ERROR pfcmd.pl(13637): Database query failed with non retryable error: Unknown column 'locationlog' in 'on clause' (errno: 1054) [SELECT `node`.`mac`, `node`.`pid`, `node`.`voip`, `node`.`bypass_vlan`, `node`.`status`, IF(ISNULL(nc.name), '', nc.name) as category, IF(ISNULL(nr.name), '', nr.name) as bypass_role, IF(node.detect_date = '0000-00-00 00:00:00', '', node.detect_date) as detect_date, IF(node.regdate = '0000-00-00 00:00:00', '', node.regdate) as regdate, IF(node.unregdate = '0000-00-00 00:00:00', '', node.unregdate) as unregdate, IF(node.lastskip = '0000-00-00 00:00:00', '', node.lastskip) as lastskip, `node`.`user_agent`, `node`.`computername`, `device_class` AS `dhcp_fingerprint`, `node`.`last_arp`, `node`.`last_dhcp`, `node`.`last_seen`, `locationlog`.`switch` AS `last_switch`, `locationlog`.`port` AS `last_port`, `locationlog`.`vlan` AS `last_vlan`, IF(ISNULL(locationlog.connection_type), '', locationlog.connection_type) as last_connection_type, `locationlog`.`dot1x_username` AS `last_dot1x_username`, `locationlog`.`ssid` AS `last_ssid`, `locationlog`.`stripped_user_name` AS `stripped_user_name`, `locationlog`.`realm` AS `realm`, `locationlog`.`switch_mac` AS `last_switch_mac`, `ip4log`.`ip` AS `last_ip`, COUNT(DISTINCT security_event.id) as nbopensecurity_events, `node`.`notes` FROM node LEFT OUTER JOIN `node_category` AS `nr` ON ( `nr`.`category_id` = `node`.`bypass_role_id` ) LEFT OUTER JOIN `node_category` AS `nc` ON ( `node`.`category_id` = `nc`.`category_id` ) LEFT OUTER JOIN security_event ON ( ( `security_event`.`status` = ? AND `node`.`mac` = `security_event`.`mac` ) ) LEFT OUTER JOIN locationlog ON ( ( `locationlog` = ? AND `node`.`mac` = `locationlog`.`mac` ) ) LEFT OUTER JOIN ip4log ON ( ( ( `ip4log` = ? OR `ip4log` > NOW() ) AND `node`.`mac` = `ip4log`.`mac` ) ) WHERE ( `node`.`tenant_id` = ? ) GROUP BY `node`.`mac`]{open, 0000-00-00 00:00:00, 0000-00-00 00:00:00, 1} (pf::dal::db_execute)
```

I'm able to reproduce on 9.3.0, a user report same issue on 9.2.0 (not verified). If error occurs on 9.2.0, it means that's not related to #4995.

**To Reproduce**

Run `pfcmd node view all`

**Expected behavior**
Display all nodes.

**Additional context**
As I mentioned on #5115, `pfcmd` logging and return could be illogical:
```console
# pfcmd node view all ; echo $?
mac|computername|pid|category|status|bypass_vlan|voip|detect_date|regdate|unregdate|last_connection_type|last_switch|last_port|last_vlan|last_ssid|last_dot1x_username|user_agent|dhcp_fingerprint|last_arp|last_dhcp|lastskip|notes
0
```

**Comment by nqb** [comment]
Fixed by d0c3319a323933128a85004718ae15fe399a5603

**Comment by nqb** [comment]
I will backport this commit to current maintenance branches.
